cypress/support/step_definitions/dictionary/createVersion/createVersion.ts

```diff
@@ -0,0 +1,57 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+import {getDictionaryId, getUser, getVersionId} from "../../../utils";
+
+Given("the user is on the dictionary page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/`);
+  cy.findByText("Versions").should("be.visible");
+});
+When("the user clicks the create new version button", () => {
+  cy.findByRole("button", { name: /Create new version/i }).click();
+});
+Then("the user should be on the create new version dialog box", () =>
+  cy.get("h2").contains("Create new version")
```

Comment:
```suggestion
  cy.findByText("Create new version").should("be.visible")
```

Reply:
Hi @ibacher, but this code will cause **Found multiple elements with the text** error. So I tried this one. Will you please suggest another idea?

Reply:
```ts
cy.findByText("Create new version", { selector: "h2" }).should("be.visible")
```